examples/SetClock/SetClock.ino

```diff
@@ -0,0 +1,62 @@
+/*
+This a simple serial port clock application using the RTC Nanoshield.
```

Comment:
Mudar para "This a simple application to set the current time for the RTC Nanoshield."